src/acat/azext_acat/custom.py

```diff
@@ -0,0 +1,538 @@
+                        "controls": [
+                            control
+                            for control in controlFamily["controls"]
+                            if control["controlStatus"]
+                            == self.ctx.args.compliance_status
+                        ],
+                    }
+                    for controlFamily in category["controlFamilies"]
+                ],
+            }
+            for category in latestSnapshot["complianceResults"][0]["categories"]
+        ]
+        latestSnapshot["complianceResults"][0]["categories"] = filteredCategories
+        return latestSnapshot
+
+
+class ListAcatReportWebhook(_AcatListReportWebhook):
+    class ListAcatReportWebhookWithDupAadToken(_AcatListReportWebhook.WebhookList):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.ListAcatReportWebhookWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class ShowAcatReportWebhook(_AcatShowReportWebhook):
+    class ShowAcatReportWebhookWithDupAadToken(_AcatShowReportWebhook.WebhookGet):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.ShowAcatReportWebhookWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class CreateAcatReportWebhook(_AcatCreateReportWebhook):
+    class CreateAcatReportWebhookWithDupAadToken(
+        _AcatCreateReportWebhook.WebhookCreateOrUpdate
+    ):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.CreateAcatReportWebhookWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        from azure.cli.core.aaz import AAZListArg, AAZStrArg, AAZStrArgFormat
+
+        args_schema.content_type._required = False
+        args_schema.content_type._registered = False
+        args_schema.events._required = False
+        args_schema.events._registered = False
+        args_schema.payload_url._required = False
+        args_schema.payload_url._registered = False
+        args_schema.send_all_events._required = False
+        args_schema.send_all_events._registered = False
+        args_schema.status._required = False
+        args_schema.status._registered = False
+        args_schema.update_webhook_key._required = False
+        args_schema.update_webhook_key._registered = False
+        args_schema.webhook_key._required = False
+        args_schema.webhook_key._registered = False
+
+        args_schema.content_type_with_default = AAZStrArg(
+            options=["--content-type"],
+            arg_group="Properties",
+            help="content type",
+            required=False,
+            enum={"application/json": "application/json"},
+            default="application/json",

```

Comment:
You can set default value in aaz-dev-tools directly https://azure.github.io/aaz-dev-tools/pages/usage/workspace-editor/#default-value-for-arguments